workers/exclusive-content/README.md

```diff
@@ -0,0 +1,92 @@
+# Exclusive Content Worker
+
+This cloudflare worker will encrypt any plain text given a payment pointer. Furthermore, it will decrypt the resulting cypher text if it can verify payment by a receipt verifier.
+
+## Testing
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
```

Comment:
do you know what the scope of this variable is? is it per-worker or per-zone?

Reply:
It is per worker